templates/pulpcore-content-apache.conf.erb

```diff
@@ -1,2 +1,8 @@
-  ProxyPass <%= scope['pulpcore::apache::content_path'] %> <%= scope['pulpcore::apache::content_url'] %>
-  ProxyPassReverse <%= scope['pulpcore::apache::content_path'] %> <%= scope['pulpcore::apache::content_url'] %>
+ProxyPass <%= scope['pulpcore::apache::content_path'] %> <%= scope['pulpcore::apache::content_url'] %>
+ProxyPassReverse <%= scope['pulpcore::apache::content_path'] %> <%= scope['pulpcore::apache::content_url'] %>
+
+<Location /pulp/content>
+RequestHeader unset X-CLIENT-CERT
+RequestHeader set X-CLIENT-CERT "%{SSL_CLIENT_CERT}s"
+</Location>
+
```

Comment:
Is it intentional/beneficial to add a new blank line to the end of these 3 templates?

Reply:
Updated..newline isn't needed here.